frontend/talentsearch/src/js/components/profile/ProfilePage/ProfilePage.stories.tsx

```diff
@@ -2,74 +2,54 @@ import React from "react";
 import { Meta, Story } from "@storybook/react";
 import { fakeUsers } from "@common/fakeData";
 import fakeExperiences from "@common/fakeData/fakeExperiences";
+import { JobLookingStatus } from "@common/api/generated";
 import { ProfilePage, ProfileForm } from "./ProfilePage";
 import { User } from "../../../api/generated";
 
-const fakeUserArray = fakeUsers(5);
+import MockGraphqlDecorator from "../../../../../.storybook/decorators/MockGraphqlDecorator";
+
+const fakeUserData = fakeUsers(1)[0];
+const fakeExperienceArray = fakeExperiences(3);
 
 export default {
   component: ProfilePage,
   title: "Profile Form",
   args: {},
+  decorators: [MockGraphqlDecorator],
+  parameters: {
+    apiResponses: {
+      getMyStatus: {
+        data: {
+          me: {
+            isProfileComplete: true,
+            jobLookingStatus: JobLookingStatus.OpenToOpportunities,
+          },
+        },
+      },
+    },
+  },
```

Comment:
This could also be set on stories like `MyStory.args` can be:

```js
export const MyStory = Template.bind({});
MyStory.parameters = {
  apiResponses: {
    getMyStatus: {
      data: {
        me: {
          isProfileComplete: true,
          jobLookingStatus: JobLookingStatus.OpenToOpportunities,
        },
      },
    },
  },
}